builder/src/non_permissioned.rs

```diff
@@ -11,7 +11,7 @@ use async_compatibility_layer::{
 use async_std::sync::{Arc, RwLock};
 use espresso_types::{
     eth_signature_key::EthKeyPair, v0_3::ChainConfig, FeeAmount, L1Client, NodeState, Payload,
-    SeqTypes, SequencerVersions, ValidatedState,
+    SeqTypes, SeqVersions, ValidatedState,
```

Comment:
Can we keep `SequencerVersions`?

Reply:
I have no preference, I think it is a consequence of that big merge. Let me switch it